### ISSUE TITLE
Git markers shown on a file not in a Git repo

### ISSUE THREAD

**Issue by borekb** [careteditor/caret#281]
For a file inside a directory that yields this:

```sh
$ git status
fatal: Not a git repository (or any of the parent directories): .git
```

I see change markers:

![image](https://cloud.githubusercontent.com/assets/101152/24514212/c2b258ca-1573-11e7-9abe-c4704afd0a9f.png)

**Comment by borekb** [comment]
Now I've started editing the file and the markers went away. (They were there when I switched to the non-Git file from a Git-tracked one.)

**Comment by astoilkov** [comment]
We think we fixed the problem in Caret 2.0.6. You could check if everything is working correctly.

Closing this issue.
